app/views/wiki/show.html.erb

```diff
@@ -64,11 +64,15 @@
     <% end %>
 
     <div class="tab-pane active" id="tab-overview">
-      <div <% unless @node.has_tag('style:wide') %>style="max-width:800px;"<% end %> id="content" class="pl-content wiki">
-        <%= raw auto_link(insert_extras(@revision.render_body), sanitize: false) %>
-      </div>
-      <% if params[:raw] == 'true' %>
-        <div style="display:none;" id="content-raw-markdown" class="pl-content wiki"><%= raw insert_extras(@revision.body_raw) %></div>
+      <% if params[:comments] == 'true' %>
```

Comment:
maybe we should rework this for a new route, like `/wiki/_name_/comments`? instead of a parameter? You could add to `config/routes.rb` and direct it to a new action that's just for comments in `comment_controller.rb`. Thoughts?